Dockerfile

```diff
@@ -13,6 +13,16 @@ RUN apt-get update \
     && docker-php-ext-configure gd --with-png-dir=/usr --with-jpeg-dir=/usr \
     && docker-php-ext-install gd mysqli opcache
 
+# install libmemcache and the php lib
+RUN apt-get update && apt-get install -y libmemcached-dev \
+    && pecl install memcached \
+    && docker-php-ext-enable memcached
+
+# install nfs support for WordPress uploads directory
+# using --no-install-recommends to prevent python install
+RUN apt-get install -y --no-install-recommends \
```

Comment:
Please consolidate all the `apt-get` into a single `RUN` statement. The `--no-install-recommends` flag is good to keep. See also https://github.com/autopilotpattern/wordpress/pull/3#discussion_r58327846

From https://github.com/10up/triton-wordpress/pull/2#discussion_r58458848